### PR TITLE
fix: support using create-cypress-tests as part of build process

### PR DESCRIPTION
**Pull request by lmiller1990**
- support importing from `create-cypress-tests` inside of `packages`
- add missing deps to `data-context`
- had to increase max zip side, can probably decrease prior to release when we remove a lot of older, unused code

Example failing step: https://app.circleci.com/pipelines/github/cypress-io/cypress/25799/workflows/adc8b63e-591b-4aa6-a4e5-119ccf9f2708/jobs/969136

### CODE REVIEW

File: scripts/binary/zip.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ const checkZipSize = function (zipPath) {
   const zipSize = filesize(stats.size, { round: 0 })
 
   console.log(`zip file size ${zipSize}`)
-  const MAX_ALLOWED_SIZE_MB = os.platform() === 'win32' ? 295 : 260
+  const MAX_ALLOWED_SIZE_MB = os.platform() === 'win32' ? 295 : 300
```
Comment (lmiller1990) [inline scripts/binary/zip.js:76]: Can decrease when we remove all the old code, prior to merging this feature branch?

File: packages/data-context/package.json [hunk 2]
```diff
@@ -14,10 +14,21 @@
...
     "@storybook/csf-tools": "^6.4.0-alpha.38",
+    "@urql/core": "2.3.1",
+    "@urql/exchange-execute": "1.1.0",
+    "@urql/exchange-graphcache": "4.3.3",
+    "chokidar": "3.5.1",
     "create-cypress-tests": "0.0.0-development",
     "cross-fetch": "^3.1.4",
     "dataloader": "^2.0.0",
+    "dedent": "^0.7.0",
+    "electron": "14.1.0",
```
Comment (tgriesser) [inline packages/data-context/package.json:25]: Must've missed when this happened. I don't think we should be depending on `electron`, if anything it should be injected from server.

**Review by tgriesser — CHANGES_REQUESTED** [review]
Saw this last week and was looking to fix before we merged #18704. One last thing needed - we need to ensure that `create-cypress-tests` is copied over to the binary builds, which it currently isn't because it doesn't start with `@cypress/`. 

It's a simple fix in [this code](https://github.com/cypress-io/cypress/blob/7ba322a12f8bd54334d995a5a8df272692f44396/scripts/binary/util/packages.js#L171-L179), but was looking to get failing tests for it before I pushed, since this isn't seeming to catch that because of how the `node_modules` resolves, thinking we need to `rm -rf packages && rm -rf npm` or something after the build, before we run the smoke tests. Will push that up in a few.

File: packages/data-context/package.json (hunk 2 above)
Comment (lmiller1990) [reply packages/data-context/package.json:25]: Yeah I agree, I thought it was a bit weird a data layer pulls in electron - we should remove that. I think @ImCesar is working around electron now, maybe he can remove the reference to electron we have in `data-context` when he is back.

**Comment by ZachJW34** [comment]
I added `create-cypress-tests` so as to use the `scanFSForAvailableDependency` function. Looks like it would have been easier to duplicate the code rather than having to setup the build steps necessary to import packages not prefixed with `@cypress` from `/npm`. Sorry to cause all the hassle!

**Pull request merged**